packages/story-editor/src/components/floatingMenu/elements/toggleUnderline.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { memo } from '@googleforcreators/react';
+import { Icons } from '@googleforcreators/design-system';
+import { __ } from '@googleforcreators/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { IconButton } from './shared';
+
+const ToggleUnderline = memo(function ToggleUnderline() {
```

Comment:
Do all these need memoizing currently? 🤔 Could you clarify why?
Isn't it a recommended practice (e.g. by Kent C Dodds) to initially not memo anything to avoid any hidden issues and add `memo` if only if really needed?

Reply:
Yes, you are right. The elements were all just a last-minute copy-paste, not thought through. Because the menus are properly memoized and are all completely static, memoizing the elements is irrelevant. The elements themselves will often re-render because of stateful hooks, but never because the parent re-renders.

Reply:
Fixed in e93628fe29.